Source/Core/Core/ConfigLoaders/IsSettingSaveable.cpp

```diff
@@ -9,6 +9,9 @@
 
 #include "Common/Config/Config.h"
 #include "Core/Config/GraphicsSettings.h"
+#ifdef USE_DISCORD_PRESENCE
+#include "UICommon/DiscordPresence.h"
```

Comment:
UI code should definitely not be in core, this is why the linkage error occurs. We don't link UICommon into Core in CMake (and it should stay that way)

Reply:
I see, I was suspecting that CMake wasn't linking UICommon. I tried to confirm that was the case in some way but my lack of knowledge in CMake didn't allow me to do that. So thanks for the confirmation.

2nd thing, IsSettingSaveable.cpp still needs MAIN_USE_DISCORD_PRESENCE. I can fix this by moving the variable from UICommon into Core.